client/src/components/Sidebar/style.scss

```diff
@@ -0,0 +1,88 @@
+$transition-time: 0.3s;
+$primary: #FC8030;
+$background: #000000BA;
+$white: #FCFCFC;
+$default: 0.5rem;
+$buttons-margin: 0.25rem;
```

Comment:
I'd prefer more descriptive names for these variables.
Also for things like $primary and $background, we should include it in the global constants file (@/constants/style)

Reply:
I'm really not sure what I was thinking 3 months ago either. I agree with the idea of making global stylesheet with base sizes and colors

Reply:
Still need to move the colors into `constants/style.scss`